api/request_source/request_source_plugin.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package nighthawk.request_source;
+
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+import "api/client/options.proto";
+
+// Configuration for DummyPluginRequestSource (plugin name: "nighthawk.dummy-request-source-plugin")
+// that does nothing
+message DummyPluginRequestSourceConfig {
```

Comment:
What is this one used for? It it only used for testing? If so, we could move it down in the file, so this isn't the first thing the reader encounters.

Assuming it is just for testing, we should use better terminology than "dummy". A test only implementation that do just one thing are usually called "Stubs".

Reply:
Understood.